examples/hello_world.py

```diff
@@ -7,3 +7,26 @@
 
 submitter = ArgoSubmitter()
 result = couler.run(submitter=submitter)
+
+# Instead of passing the submitter each time, you can just set it as default!
+ArgoSubmitter.set_default(submitter)
+
+couler.run_container(
+    image="docker/whalesay",
+    command=["cowsay"],
+    args=["hello world with default submitter"],
+)
+
+result_with_default_submitter = (
+    couler.run()
+)  # no submitter need be passed now!
```

Comment:
Perhaps move this example to a separate file since hello world example is meant to be simple and concise

Reply:
agreed